Resources/Prototypes/_Omu/Entities/Clothing/Modsuit/modsuit.yml

```
@@ -0,0 +1,491 @@
+        Dismemberment: 0.1
+        OrganDamage: 0.1
+        BoneDamage: 0.1
+        VeinsDamage: 0
+        NerveDamage: 0
+      modifiers:
+        coefficients:
+          Blunt: 0.9
+          Slash: 0.9
+          Piercing: 0.9
+    - type: ToggleClothing
+      action: ActionToggleMagboots
+    - type: ComponentToggler
+      components:
+      - type: NoSlip
+    - type: Magboots
+      enabledPrefix: null
+    - type: ClothingSpeedModifier
+      walkModifier: 0.9
+      sprintModifier: 0.9
+
+
+# Phobos Modsuit
+
+- type: entity
+  parent: [ClothingModsuitStandard, BaseSyndicateContraband]
+  id: EliteModsuitBackpack
+  name: CSI-ECU-02 "Phobos" Modsuit Control
+  description: A controller for the "Phobos" modsuit.
+  components:
+    - type: Sprite
+      sprite: _Omu/Clothing/Modsuit/syndicateelitemodsuit.rsi
+    - type: ExplosionResistance
+      damageCoefficient: 0.6
+    - type: ToggleableClothing
+      clothingPrototypes:
+        head: ClothingModsuitHelmetEliteSyndicateModsuit
+        gloves: ClothingModsuitGauntletsEliteSyndicateModsuit
+        outerClothing: ClothingModsuitChestplateEliteSyndicateModsuit
+        shoes: ClothingModsuitBootsEliteSyndicateModsuit
+    - type: ItemSlots
+      slots:
+        cell_slot:
+          name: power-cell-slot-component-slot-name-default
+          startingItem: PowerCellHigh
+          whitelist:
+            components:
+              - PowerCell
+    - type: Storage
+      maxItemSize: Huge
+      grid:
+        - 0,0,7,4
+    - type: SealableClothingVisuals
+      clothingVisuals:
+        back:
+        - state: equipped-BACKPACK-sealed
+        - state: equipped-BACKPACK-sealed_glow
+          shader: unshaded
+
+- type: entity
+  parent: [ClothingModsuitGauntletsStandard, BaseSyndicateContraband]
+  id: ClothingModsuitGauntletsEliteSyndicateModsuit
+  name: CSI-EM-02 "Phobos" Modsuit Gauntlets
+  description: A pair of gauntlets made for the "Phobos" modsuit.
+  categories: [HideSpawnMenu]
+  components:
+    - type: Sprite
+      sprite: _Omu/Clothing/Modsuit/syndicateelitemodsuit.rsi
+    - type: SealableClothing
+      sealUpPopup: sealable-clothing-seal-up-gauntlets
+      sealDownPopup: sealable-clothing-seal-down-gauntlets
+    - type: SealableClothingVisuals
+      visualLayers:
+        gloves:
+        - state: equipped-HAND-sealed
+    - type: Armor
+      modifiers:
+        coefficients:
+          Blunt: 0.9
+          Slash: 0.9
+          Piercing: 0.9
+          Shock: 0.75
+          Heat: 0.70
+    - type: Insulated
+    - type: Fiber
+      fiberMaterial: fibers-modular
+      fiberColor: fibers-black
+    - type: DamageOnInteractProtection
+      damageProtection:
+        flatReductions:
+          Heat: 10
+    - type: Tag
+      tags:
+      - PlasmamanSafe
+    - type: IgniteFromGasImmunity
+      parts:
+        - LeftHand
+        - RightHand
+
+- type: entity
+  parent: [ClothingModsuitHelmetStandard, ShowSecurityIcons, BaseSyndicateContraband]
```

```suggestion
  parent: [ClothingModsuitHelmetStandard, ShowSecurityIcons, BaseSyndicateContraband, ShowHealthRelatedIcons]
```
Easier to just parent from this for the medical hud bits